app/src/main/java/com/example/dharan1011/popular_movie_app/Utils/APIService.java

```diff
@@ -19,7 +19,7 @@
 
 public interface APIService {
     String BASE_URL = "http://api.themoviedb.org/3/";
-    String API_KEY = "4182aa25bab27d06344e404f65c4ae76";
+    String API_KEY = "PUT_YOUR_KEY_HERE";
```

Comment:
Good that you have remove the api_key. You might have added to TODO so that it would be easy for other developers to find where to put the api key